must-gather/collection-scripts/gather

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 
+
+function is_hypershift() {
+  control_plane_topology=$(/usr/bin/oc get infrastructure cluster -ojson | jq .status.controlPlaneTopology)
```

Comment:
we can also use kubectl advanced formatting and very likely get rid of the `jq` part, but AFAIR `jq` is already installed, so good enough for now